source/features/last-notification-page-button.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager';
+import looseParseInt from '../helpers/loose-parse-int';
+
+function init(): void {
+	const nextButtonURL = new URL(select('a[data-hotkey="ArrowRight"]')!.href);
+	const nextButtonSearch = new URLSearchParams(nextButtonURL.search);
+	const lastNotificationPageNode = select('.js-notifications-list-paginator-counts')!.lastChild!;
+	const lastNotificationPageNumber = looseParseInt(lastNotificationPageNode);
+	const lastCursor = Math.floor(lastNotificationPageNumber / 50) * 50;
+	nextButtonSearch.set('after', btoa('cursor:' + String(lastCursor)));
+	nextButtonURL.search = nextButtonSearch.toString();
```

Comment:
I think you can set the raw search string directly onto the anchor, without using nextButtonURL at all ever:
```js
const nextButtonSearch = new URLSearchParams(nextButton.search)

// etc

<a href={String(nextButtonSearch)}>

Reply:
This will not work `{String(nextButtonSearch) => after=Y3Vyc29yOjEwNTA%3D&query=is%3Adone`